hack/local-development/remote-garden/generate-certs

```diff
@@ -64,5 +64,4 @@ openssl x509 -req -in "$CERTS_DIR/$CLIENT_NAME.csr" -CA "$CERTS_DIR/$CA_NAME.crt
 
 # Clean up after we're done.
 rm "$CERTS_DIR/$SERVER_NAME.csr" "$CERTS_DIR/$CLIENT_NAME.csr"
-rm "$CERTS_DIR/$CA_NAME.srl"
```

Comment:
Why is this line removed? At least on my setup, such a file is created by `openssl` in one of the previous steps, and without this line it is still there in `$CERTS_DIR` after everything should have been cleaned up. If the behavior of `openssl` is different and this breaks on your side, please add a check if the file exists before removeing it.

Reply:
thanks